### ISSUE TITLE
memleak on RAI_ModelSerializeTFLite  on *buffer = RedisModule_Calloc(tflitebuffer->len, sizeof(char)); 

### ISSUE THREAD

**Issue by filipecosta90** [RedisAI/RedisAI#287]
leak on 
``` c
*buffer = RedisModule_Calloc(tflitebuffer->len, sizeof(char)); 
```

```
=28890== 3,284,088 bytes in 1 blocks are definitely lost in loss record 132,299 of 132,299
==28890==    at 0x4C31B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==28890==    by 0x144DB6: zcalloc (zmalloc.c:131)
==28890==    by 0x1C6FAC: RM_Calloc (module.c:335)
==28890==    by 0x2428CF81: RAI_ModelSerializeTFLite (tflite.c:126)
==28890==    by 0x67E79E7: RAI_ModelSerialize (model.c:389)
==28890==    by 0x67D823A: RedisAI_ModelGet_RedisCommand (redisai.c:717)
==28890==    by 0x1C76BE: RedisModuleCommandDispatcher (module.c:575)
==28890==    by 0x13D315: call (server.c:2468)
==28890==    by 0x13DFCA: processCommand (server.c:2763)
==28890==    by 0x14F086: processInputBuffer (networking.c:1470)
==28890==    by 0x14F1DB: processInputBufferAndReplicate (networking.c:1505)
==28890==    by 0x14F551: readQueryFromClient (networking.c:1587)
```

**Comment by lantiga** [comment]
Solved in #289 (see 800695d)

**Comment by filipecosta90** [comment]
confirming that leak was solved
